benchmark/utils/DriverUtils.swift

```diff
@@ -167,28 +161,23 @@ struct TestConfig {
   }
 
   mutating func findTestsToRun() {
-    var i = 1
-    for benchName in precommitTests.keys.sorted() {
-      tests.append(Test(name: benchName, n: i, f: precommitTests[benchName]!))
-      i += 1
-    }
-    for benchName in otherTests.keys.sorted() {
-      tests.append(Test(name: benchName, n: i, f: otherTests[benchName]!))
-      i += 1
-    }
-    for benchName in stringTests.keys.sorted() {
-      tests.append(Test(name: benchName, n: i, f: stringTests[benchName]!))
-      i += 1
-    }
-    for i in 0..<tests.count {
-      if onlyPrecommit && precommitTests[tests[i].name] == nil {
-        tests[i].run = false
-      }
-      if !filters.isEmpty &&
-         !filters.contains(String(tests[i].index)) &&
-         !filters.contains(tests[i].name) {
-        tests[i].run = false
+    let allTests = [precommitTests, otherTests, stringTests]
+      .map { dictionary -> [(key: String, value: (Int)-> ())] in
+        Array(dictionary).sorted { $0.key < $1.key } } // by name
+      .joined()
+
+    let included: Set<String>
+      if !filters.isEmpty  { included = Set(filters) } else {
```

Comment:
Because I had to manually space indent by 2 spaces, like an animal… and made a mistake.

Would you provide the rough consensus @practicalswift seeks to fix formatting in this file? [gardening]